### PR TITLE
[CUBRIDQA-70] merge ctp parameter refine result into develop

### PR DESCRIPTION
**Pull request by supler**
@jinmeihua @mandy2016 @slamdunkorchid @jiyanping 

So far we have finished CTP parameters enhancement, please review the below points:
1. README.md
2. All parameter names are expected as our discussion
3. All codes changed meet the standards of open source

thanks

### CODE REVIEW

**Comment by slamdunkorchid** [comment]
@supler 
Regarding more explanation for parameters setting, please refer to CTP/conf/sql.conf for SQL, CTP/conf/medium.conf for Medium and CTP/conf/sql_by_cci.conf for SQL_By_CCI

the link for  CTP/conf/sql_by_cci.conf  is invalid

File: CTP/README.md [hunk 1]
```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
-	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
-	
-	  ```
-	   // on controller node
+        * For local instance test
+	   * Prepare one account to use for controller and test instance
```
Comment (slamdunkorchid) [inline CTP/README.md]: confuse:
common use of one account , or separation use of one account

File: CTP/README.md [hunk 2]
```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
...
+	   * CTP and test scenario should be checked out on it, and configure JAVA_HOME
+	   ```
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+	   ```
+	   * **Example** ``shell.conf`` for local instance, scenario and test build:
+	   ```
+	   # Configure parameters for cubrid.conf following the format outlined below, ie, default.cubrid.cubrid_port_id=1523
+	   default.cubrid.<property>=<property value> 
+	   # Configure parameters for cubrid_broker.conf following the format outlined below
+	   default.broker1.<property>=<property value>
```
Comment (jiyanping) [inline CTP/README.md]: It's better to point out that broker1 represent '%query_editor' and broker2 represent '%BROKER1' in cubrid_broker.conf

File: CTP/README.md [hunk 3]
```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
```
Comment (jiyanping) [inline CTP/README.md]: As is,
It's not required that you execute build for CTP
To be,
It's not required that you recompile CTP

Comment (jiyanping) [inline CTP/README.md]: As is,
When you want to write your own test case
To be,
When you plan to write your own test case

Comment (jiyanping) [inline CTP/README.md]: As is,
verify the data synchronization 
To be
verify the data replication

**Comment by slamdunkorchid** [comment]
@supler 
for shell/Isolation/ha repl 
whether it need to make some basic concepts more clearly
eg:

The Basics

how to execute test for multiple instances test? It use this structure one controller node and multiple test nodes.


 A controller
         is the machine you deploy your test tools,prepare conf file,start a test task.
A test node
        is  the machine where the test actual be executed

**Comment by slamdunkorchid** [comment]
@supler 
# On controller node
JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)

# On test instance node
CTP_HOME (e.g., export CTP_HOME=$HOME/CTP) 
JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)


it is easy to misleading users that CTP_HOME not need to configured on  controller node

**Comment by slamdunkorchid** [comment]
@supler 
In HA Replication
 As is,
env.instance1.broker.BROKER_PORT=35000
To be
env.instance1.broker1.BROKER_PORT=35000

**Comment by jiyanping** [comment]
@supler 
You can find the generated jar files common/lib/cubridqa-common.jar, sql/lib/cubridqa-cqt.jar, common/sched/lib/cubridqa-scheduler.jar, shell/init_path/commonforjdbc.jar, shell/lib/cubridqa-shell.jar, isolation/lib/cubridqa-isolation.jar and ha_repl/lib/cubridqa-ha_repl.jar

Is this meaningful to user?

**Comment by slamdunkorchid** [comment]
@supler 
In HA Replication

- main_snapshot.properties saves all values of parameters configured during testing

Now:
ha_repl can not find main_snapshot.properties  files

**Comment by jinmeihua** [comment]
CTP/README.md

As is,( feeling a little wordy...)
This Quick Start is only for user for reference about how to use CTP to start SQL test quickly. But CTP supports more categories testing than this section mentioned, such as Shell, CCI, HA Shell, Isolation, HA Replication, SQL_By_CCI, Jdbc and so on. Regarding more information please refer to the related sections
To be, 
This Quick Start is only about how to use CTP to start SQL test quickly. For more test categories such as Shell, CCI, HA Shell, Isolation, HA Replication, SQL_By_CCI and Jdbc , please refer to the related sections

As is,
For Medium test, data_file must be set to the path of the initial data file
To be, (remind users to add the data_file to [sql] section.)
For Medium test, data_file must be set to the path of the initial data file in the SQL section.

**Comment by slamdunkorchid** [comment]
@supler 
 in **How To Write Testcase**
 SHELL

. $init_path/init.sh
whther it is need to tell user $init_path value

File: CTP/README.md [hunk 4]
```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
...
+	   scenario=${HOME}/cubrid-testcases/shell
+	   ```
+	   ```
+	   # Define the URL of test build which will be used to test.
+	   # If this parameter is not set or commented out, CTP will execute testing without build installation.
+	   cubrid_download_url=http://127.0.0.1/download/CUBRID-10.1.0.6929-b049ba5-Linux.x86_64.sh
+	   ```
+        * For multiple instances test
+	   * Prepare at least two accounts for the multiple test instances (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
+	   * Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
```
Comment (jinmeihua) [inline CTP/README.md]: Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
Check out scenario for each instances.

**Comment by fanzaiqiang** [comment]
@supler 

Please go ahead to merge it. Thanks a lot.

**Pull request merged**